tt_metal/impl/dispatch/kernels/cq_dispatch.cpp

```diff
@@ -125,6 +141,123 @@ void get_dispatch_cb_page() {
     cb_fence += n_pages * dispatch_cb_page_size;
 }
 
+FORCE_INLINE volatile uint32_t* get_cq_completion_read_ptr() {
+    return reinterpret_cast<volatile uint32_t*>(CQ_COMPLETION_READ_PTR);
+}
+
+FORCE_INLINE volatile uint32_t* get_cq_completion_write_ptr() {
+    return reinterpret_cast<volatile uint32_t*>(CQ_COMPLETION_WRITE_PTR);
+}
+
+FORCE_INLINE
+void completion_queue_reserve_back(uint32_t num_pages) {
+    DEBUG_STATUS('N', 'Q', 'R', 'B', 'W');
```

Comment:
debug status is limited to 4 characters, last get dropped

Reply:
This was actually copied from the existing FD 1 function so I guess no one ever caught this issue 🙂 . Will fix